slick/src/main/scala/slick/jdbc/JdbcDataSource.scala

```diff
@@ -14,12 +15,23 @@ import slick.SlickException
   * similar to a `javax.sql.DataSource` but simpler. Unlike [[JdbcBackend.DatabaseDef]] it is not a
   * part of the backend cake. This trait defines the SPI for 3rd-party connection pool support. */
 trait JdbcDataSource extends Closeable {
+
+  var pausable: Option[Pausable] = _
+
+  def registerPausable(p: Pausable) = {
+    pausable = Option(p)
```

Comment:
Probably more idiomatic way to do it would be to initially assign `None` to it, at use `Some` here.
Is there any reason wrote it in this way (null and Option)?